explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -537,6 +543,21 @@ export const buildBatchCorrectionWarning = (): React.ComponentProps<
   };
 };
 
+/**
+ * Build props for the biological network cell component.
+ * @param projectsResponse - Response model return from projects API.
+ * @returns Props to be used for the cell.
+ */
+export const buildBioNetwork = (
+  projectsResponse: ProjectsResponse
+): React.ComponentProps<typeof BioNetworkCell> => {
+  const project = getProjectResponse(projectsResponse);
+  return {
+    label: getPluralizedMetadataLabel(METADATA_KEY.BIONETWORK_NAME),
+    networkKeys: project.bionetworkName,
```

Comment:
Let's process the networkKeys in the `viewModelBuilder` to only pass to the component a valid list of keys.